### DISCUSSION TITLE
Add option to PDF Compare Studio to go to specific page

### DISCUSSION THREAD

**Discussion by Alex-0529** [red6/pdfcompare#143] [Q&A]
I think it may be a helpful option to be able to go to specific page. Like it is in Acrobat reader for example.
![image](https://github.com/red6/pdfcompare/assets/50147319/a7fb0611-c986-4708-80b0-cfc97276b1b4)

it will be great if all 3 PDF documents (Expected, Actual, Result) will be in sync on page position

**Comment by finsterwalder** [comment]
Yes, that's a nice idea. But I will not work on it.
Feel free to submit a merge request.
